Dockerfile

```diff
@@ -1,12 +1,20 @@
 FROM elifesciences/python:b22fbad8f66100d88cb4bd7c7d092b376a9e09bc
 
+ENV LANG=C.UTF-8
```

Comment:
click complained.

Reply:
Not sure who click is, but I've seen this before e.g. in bot-lax-adaptor as environments like servers start from an empty set of environment variables that sometimes are instead set locally

Reply:
http://click.pocoo.org/5/python3/#python-3-surrogate-handling